packages/core/src/__tests__/document.test.ts

```diff
@@ -11,6 +11,7 @@ import { TileDoctype, TileParams, TileDoctypeHandler } from "@ceramicnetwork/doc
 import { PinStore } from "../store/pin-store";
 import { LevelStateStore } from "../store/level-state-store";
 import { DID } from "dids"
+import { sha256 } from 'js-sha256'
```

Comment:
Not super important yet, but we should be using `@stablelib/sha256` everywhere.

Reply:
Ack.  Decided to leave it as-is in the interest of time since this is test code and there is other test code already working this way (which I copied here).  We can change it all in one fell swoop when needed.